cmd/cephcsi.go

```diff
@@ -75,6 +75,7 @@ func init() {
 	flag.IntVar(&conf.PidLimit, "pidlimit", 0, "the PID limit to configure through cgroups")
 	flag.BoolVar(&conf.IsControllerServer, "controllerserver", false, "start cephcsi controller server")
 	flag.BoolVar(&conf.IsNodeServer, "nodeserver", false, "start cephcsi node server")
+	flag.BoolVar(&conf.EnableTopology, "enable-topology", false, "enable for topology aware provisioning")
```

Comment:
As it is currently enabled by default, users that upgrade will suddenly see this disabled. In Kubernetes the default is to use topology aware provisioning too, is there a good reason to have it disabled by default here?

Reply:
I was thinking since ceph-csi had feature gate topology disabled by default (`--feature-gates=Topology=false` https://github.com/ceph/ceph-csi/pull/4660/commits/4ecb074fac55e77db37dbce384594d09936a6e03) so keep it disabled by default now?

Reply:
Let's add a release note mentioning the new flag.

Reply:
The `csi-provisioner` container-image enables the feature flag by default, the feature is GA. I think it makes sense to enable it by default in Ceph-CSI too? It will not be used if not configured, that should not be problematic?